cypress/integration/taskCloneTableColumn.spec.js

```diff
@@ -1,33 +1,24 @@
 'use strict';
 
 describe('Clone table app', () => {
-  before(() => {
+  beforeEach(() => {
     cy.visit('/');
+    cy.get('th:nth-child(2)').debug();
   });
 
   it('should only change number of columns in original table', () => {
     cy.get('table').children().should('have.length', 3);
-    cy.get('thead > tr').children().should('have.length', 6);
+    cy.get('thead > tr > th').should('have.length', 6);
     cy.get('thead').children().should('have.length', 1);
     cy.get('tbody').children().should('have.length', 12);
     cy.get('tfoot').children().should('have.length', 1);
   });
 
-  it('should copy second column and paste between fourth and sixth', () => {
-    cy.get('th:nth-child(2)').then(($th2) => {
-      const txt = $th2.text();
+  it('should copy second column and paste before the fifth one', () => {
+    cy.get('thead th').eq(1).should('have.text', 'Position');
+    cy.get('thead th').eq(4).should('have.text', 'Position');
```

Comment:
This assertion seems to be incorrect because it's checking that both the second and fifth header have the same text 'Position', which is not the expected behavior when cloning a column. The text 'Position' should only be present in the header of the cloned column, not in both.